### PR TITLE
Hr worked days hourly rate

### PR DESCRIPTION
**Pull request by coleste**

### CODE REVIEW

File: hr_worked_days_hourly_rate/tests/test_hr_worked_days.py [hunk 1]
```diff
@@ -25,68 +25,47 @@
...
-        self.payslip_model.unlink(
-            self.cr, self.uid, [self.payslip_id], context=self.context)
-        self.contract_model.unlink(
-            self.cr, self.uid, [self.contract_id], context=self.context)
-        self.employee_model.unlink(
-            self.cr, self.uid, [self.employee_id], context=self.context)
+        self.payslip_id.unlink()
+        self.contract_id.unlink()
+        self.employee_id.unlink()
         super(test_worked_days, self).tearDown()
```
Comment (ghost) [inline hr_worked_days_hourly_rate/tests/test_hr_worked_days.py]: You may remove the tear down method. It is not required.

File: hr_worked_days_hourly_rate/__init__.py [hunk 2]
```diff
@@ -19,6 +19,4 @@
 #
 ##############################################################################
 
-from . import (
-    hr_payslip_worked_days,
-)
+import models
```
Comment (ghost) [inline hr_worked_days_hourly_rate/__init__.py]: use relative import:
from . import models

**Comment by ghost** [comment]
@coleste travis is failing because of PEP8 errors. Once you fix this plus my 2 comments, then this will be ready to merge.

**Comment by max3903** [comment]
@coleste CLA is fixed. I was missing your Github login.

@rschnapka Any chance to sign the CLA ?

File: hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+#
+##############################################################################
+
+from openerp import fields, models
+
+
+class HrPayslipWorkedDays(models.Model):
+    _inherit = 'hr.payslip.worked_days'
+
+    def _compute_total(self):
```
Comment (pedrobaeza) [inline hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py]: You should put @api.one and @api.depends for the compute field.

File: hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    hourly_rate = fields.Float(
+        'Hourly Rate',
+        default=0,
+        help="""
+The employee's standard hourly rate for one hour of work.
+Example, 25 Euros per hour.
+        """)
+    rate = fields.Float(
+        'Rate (%)',
```
Comment (pedrobaeza) [inline hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py:34]: Don't make this ugly line split. It's not needed for the correct read of the field.

File: hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        default=100,
+        help="""
```
Comment (pedrobaeza) [inline hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py]: Putting triple quota you introduce break lines on the translatable texts, making them ugly. Use instead:

```
help="A line very long that "
     "needs two lines"
```

File: hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+The rate by which to multiply the standard hourly rate.
+Example, an overtime hour could be paid the standard rate multiplied by 150%.
+        """)
+
+    # When a worked day has a number of hours and an hourly rate,
+    # it is necessary to have a date interval,
+    # because hourly rates are likely to change over the time.
+    date_from = fields.Date('Date From', default=fields.Date.context_today)
+    date_to = fields.Date('Date To', default=fields.Date.context_today)
+    total = fields.Float(string='Total', compute=_compute_total)
```
Comment (pedrobaeza) [inline hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py]: Put compute method between quotes to allow easy inheritance

**Comment by ghost** [comment]
:+1:

**Comment by ghost** [comment]
@pedrobaeza you may remove the needs fixing label

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
One last thing before merging: please extract description to a README.rst file using the OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by ghost** [comment]
@coleste are you still working on this PR ?

**Comment by feketemihai** [comment]
Work in progress in #188 , contribution and tests grabbed from this pr.

**Pull request closed without merging**